components/datadog/apps/aspnetsample/ecs.go

```diff
@@ -0,0 +1,117 @@
+package aspnetsample
+
+import (
+	"github.com/DataDog/test-infra-definitions/common/config"
+	"github.com/DataDog/test-infra-definitions/common/utils"
+	fakeintakeComp "github.com/DataDog/test-infra-definitions/components/datadog/fakeintake"
+	"github.com/DataDog/test-infra-definitions/resources/aws"
+	ecsClient "github.com/DataDog/test-infra-definitions/resources/aws/ecs"
+	classicECS "github.com/pulumi/pulumi-aws/sdk/v6/go/aws/ecs"
+	"github.com/pulumi/pulumi-awsx/sdk/v2/go/awsx/ecs"
+	"github.com/pulumi/pulumi-awsx/sdk/v2/go/awsx/lb"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+type EcsFargateComponent struct {
+	pulumi.ResourceState
+}
+
+func FargateAppDefinition(e aws.Environment, clusterArn pulumi.StringInput, apiKeySSMParamName pulumi.StringInput, fakeIntake *fakeintakeComp.Fakeintake, opts ...pulumi.ResourceOption) (*EcsFargateComponent, error) {
+	namer := e.Namer.WithPrefix("aspnetsample").WithPrefix("fg")
+
+	opts = append(opts, e.WithProviders(config.ProviderAWS, config.ProviderAWSX))
+
+	EcsFargateComponent := &EcsFargateComponent{}
+	if err := e.Ctx.RegisterComponentResource("dd:apps", namer.ResourceName("grp"), EcsFargateComponent, opts...); err != nil {
+		return nil, err
+	}
+
+	opts = append(opts, pulumi.Parent(EcsFargateComponent))
+
+	nlb, err := lb.NewNetworkLoadBalancer(e.Ctx, namer.ResourceName("lb"), &lb.NetworkLoadBalancerArgs{
```

Comment:
Why do you need an LB?

Reply:
I was thinking of adding a query app too in another PR.

Reply:
I removed the load balancer so we can add it in another PR if needed